package-lock.json

```diff
@@ -0,0 +1,6300 @@
+{
```

Comment:
Is `package-lock.json` something that should be part of the repository? I don't think we add the lock file to repositories.

Reply:
Generally yes, it's supposed to be kept in VCS. It's purpose is to lead to predictable environment during build and deploy (which also leads to the same transpiled code on different machines).